docs/intro/basic-tutorial.rst

```diff
@@ -511,7 +511,15 @@ and store the :py:class:`web_poet.rules.ApplyRule` for you. All of the
     # rules from other packages. Otherwise, it can be omitted.
     # More info about this caveat on web-poet docs.
     consume_modules("external_package_A", "another_ext_package.lib")
-    SCRAPY_POET_RULES = default_registry.get_rules()
```

Comment:
The text above tells

> All of the
:py:class:`web_poet.rules.ApplyRule` rules could then be simply read as:

But the example no longer reads the rules.

Reply:
Good catch, I've cleaned up the paragraphs above.